style.css

```diff
@@ -31,6 +31,15 @@ body, .entry-content {
 	font-family: var( --font-body );
 }
 
+.entry-content h1, 
+.entry-content h2, 
+.entry-content h3, 
+.entry-content h4, 
+.entry-content h5,
+.entry-content h6 {
+	font-family: var( --font-header );
+}
```

Comment:
How does this differ from the plain elements below? Also, note that h3 uses a different font.

Reply:
TwentyTwenty targets headings at a higher specificity:
<img width="543" alt="Screen Shot 2020-04-21 at 10 04 57 PM" src="https://user-images.githubusercontent.com/5375500/79932761-416f8600-841c-11ea-927e-90d95243c381.png">

These rules would override it.

> Also, note that h3 uses a different font.

Ahh got it 👍